### PR TITLE
Feat/avatar badge

### PR DESCRIPTION
**Pull request by Aditi0505**
image
image
image
image
image
image

### CODE REVIEW

File: docs/site.html [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+          <img src="/images/taylorSwift.jpg" alt="avatar" class="avatar avatar-lg">
+        </div>
+        <iframe
+          src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=duotone-dark&wt=none&l=css&width=776&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522%252Fimages%252FtaylorSwift.jpg%2522%2520alt%253D%2522avatar%2522%2520class%253D%2522avatar%2520avatar-xs%2522%253E%250A%253Cimg%2520src%253D%2522%252Fimages%252FtaylorSwift.jpg%2522%2520alt%253D%2522avatar%2522%2520class%253D%2522avatar%2520avatar-sm%2522%253E%250A%253Cimg%2520src%253D%2522%252Fimages%252FtaylorSwift.jpg%2522%2520alt%253D%2522avatar%2522%2520class%253D%2522avatar%2520avatar-md%2522%253E%250A%253Cimg%2520src%253D%2522%252Fimages%252FtaylorSwift.jpg%2522%2520alt%253D%2522avatar%2522%2520class%253D%2522avatar%2520avatar-lg%2522%253E"
+          style="width: 100%; height: 243px; border:0; transform: scale(1); overflow:hidden;"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
```
Comment (anupkgurung) [inline docs/site.html:112]: here instead of having fixed height and width you can add the iframe inside a div and give them width of 100%.

**Review by prakashsakari — APPROVED** [review]
Looks good to me

File: style.css [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+}
+.cta-install:hover {
+  background-color: var(--hover);
+  color: var(--primary-color);
+}
+.title {
+  color: var(--accent1);
+}
+
+.fas {
```
Comment (anupkgurung) [inline style.css:212]: fas class name can be improved not sure if fas is being used. Few sugg( f-sm or font-sm, -md, lg).

**Review by anupkgurung — APPROVED** [review]
few  suggestion added. Rest is good !!

File: style.css (hunk 2 above)
Comment (Aditi0505) [reply style.css:212]: fas is font-awesome class name by default. So I am directly using it.
let me know if I should continue using it or add a new class to it.

File: docs/site.html (hunk 1 above)
Comment (Aditi0505) [reply docs/site.html:112]: Sure. I am facing issue with iframe. planning to change this in all places. thanks for suggesting this.

File: components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  position: absolute;
+  background-color: var(--accent7);
+  width: var(--md-size);
+  height: var(--md-size);
+  border-radius: 50%;
+  top: -5px;
+  right: -10px;
+  font-size: var(--sm-size);
+  font-weight: var(--bold);
+  color: var(--primary-color);
```
Comment (Naveenchandar) [inline components/badge/badge.css:50]: Lots of Style property has been repeated in **badge-number** , **large-badge-number** ,  **badge-large** ,**badge-label-large**.  Please use common class for it.

File: docs/docs-style.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--primary-color);
+  align-items: center;
+  border-right: 1px solid var(--accent2);
+  border-bottom: 1px solid var(--accent2);
```
Comment (Naveenchandar) [inline docs/docs-style.css:17]: Please use shorthand property for border

File: docs/docs-style.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  position: fixed;
+  top: 60px;
+  height: 90vh;
+  padding-top: var(--xs-size);
+}
+.sidebar-title {
+  text-align: left;
+  color: var(--accent1);
+  margin-left: var(--sm-size);
+  margin-right: var(--lg-size);
```
Comment (Naveenchandar) [inline docs/docs-style.css:27]: Please use shorthand property for margin

File: components/badge/badge.css (hunk 3 above)
Comment (Aditi0505) [reply components/badge/badge.css:50]: Sure! Thanks for suggesting this, I will do the needful.

File: docs/docs-style.css (hunk 4 above)
Comment (Aditi0505) [reply docs/docs-style.css:17]: Sure, I will work on this.

File: docs/docs-style.css (hunk 5 above)
Comment (Aditi0505) [reply docs/docs-style.css:27]: Sure, I will work on this.

File: docs/docs-style.css [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+.badge-icon-example,
+.badge-avatar-example,
+.badge-button-example,
+.badge-label-example {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid var(--accent2);
+  padding: var(--sm-size);
+  gap: var(--md-size);
```
Comment (Naveenchandar) [inline docs/docs-style.css:91]: If these are some common properties, you add all in one common class

File: docs/docs-style.css [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+  justify-content: space-between;
+  text-align: left;
+}
+.items {
+  padding: var(--sm-size);
+}
+.item-link {
+  color: var(--cta);
+  font-size: var(--md-size);
+  font-family: var(--body-font);
```
Comment (Naveenchandar) [inline docs/docs-style.css:43]: Please use shorthand property

File: style.css (hunk 2 above)
Comment (anupkgurung) [reply style.css:212]: oh okay than it's fine. I had no idea thanks !!

**Pull request merged**